Runtime/Extensions/StringExtensions.cs

```diff
@@ -95,6 +95,12 @@ public static string PathFromURI(this string value)
         public static string FilenameFromURI(this string value)
             => value.Substring(value.BackSlashes().LastIndexOf(Path.DirectorySeparatorChar) + 1, value.Length - value.BackSlashes().LastIndexOf(Path.DirectorySeparatorChar) - 1);
 
+        /// <summary>
+        /// Returns a filename without its extenion
+        /// </summary>
+        public static string FilenameWithoutExtension(this string value)
```

Comment:
Why not use `Path.GetFileNameWithoutExtension`?

Reply:
Updated